keras/applications/regnet.py

```diff
@@ -0,0 +1,1349 @@
+# Copyright 2021 Aditya Kane. All Rights Reserved.
```

Comment:
This should say "# Copyright 2021 The TensorFlow Authors. All Rights Reserved." (you are a TensorFlow Author)